CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+Please note we have a [Code of Conduct](https://openactive.io/public-openactive-w3c/code-of-conduct/), please follow it in all your interactions with the project.
+
+## Reporting Issues & Requesting Features
+
+If you notice something wrong with this code, or think there's something that it should do, please open a GitHub Issue. Before creating a new issue check to see if the issue already exists. If not, then please do report it. 
+
+## Code Quality
+
+Spontaneous bug fixes and incremental feature PRs are welcome! However, before making substantial changes to this repository, we encourage you to first discuss the change you wish to make via an issue, Slack, or any other method with the owners of this repository.
```

Comment:
"Slack, or any other method with the owners of this repository." Some links for this could be good